src/mendeleyMain.vb

```diff
@@ -692,6 +692,33 @@ EndOfSub:
     uiDisabled = False
 End Sub
 
+Sub Remove_Bookmark()
+On Error GoTo ErrorHandler
+     Dim mBookmarks,rbTemp
+     Dim j As Long
+     Dim args1(0) as new com.sun.star.beans.PropertyValue
+     Dim document   as object
+     Dim dispatcher as object
+
+     document   = ThisComponent.CurrentController.Frame
+     dispatcher = createUnoService("com.sun.star.frame.DispatchHelper")
+     mBookmarks = thisComponent.Bookmarks.ElementNames
+    If UBound(mBookmarks) = 0 then
```

Comment:
I don't think that this If is needed - the below "For j = 0 To UBound(mBookmarks)" will not iterate if mBookmarks is empty.